src/components/createCourse/CreateCourse.types.ts

```diff
@@ -0,0 +1,6 @@
+export interface CreateCourseProps {
+  setCoursesList: any;

```

Comment:
Let's try to avoid `any`. If you struggle to define a type for hook's functions you can hover over it and see the type definition in you IDE.